api/conf/schema.json

```diff
@@ -1 +1 @@
iple header use ',' to split. default: *.","default":"*","type":"string"},"allow_methods":{"description":"you can use '*' to allow all methods when no credentials and '**','**' to allow forcefully(it will bring some security risks, be carefully),multiple method use ',' to split. default: *.","default":"*","type":"string"},"max_age":{"description":"maximum number of seconds the results can be cached.-1 mean no cached,the max value is depend on browser,more detail plz check MDN. default: 5.","default":5,"type":"integer"},"allow_credential":{"description":"allow client append crendential. according to CORS specification,if you set this option to 'true', you can not use '*' for other options.","default":false,"type":"boolean"},"expose_headers":{"description":"you can use '*' to expose all header when no credentials,multiple header use ',' to split. default: *.","default":"*","type":"string"}}},"example-plugin":{"type":"object","properties":{"s":{"type":"string"},"t":{"type":"array","minItems":1},"i":{"type":"number","minimum":0},"port":{"type":"integer"},"ip":{"type":"string"}},"required":["i"]},"limit-conn":{"type":"object","properties":{"conn":{"type":"integer","minimum":0},"default_conn_delay":{"type":"number","minimum":0},"rejected_code":{"type":"integer","minimum":200,"default":503},"burst":{"type":"integer","minimum":0},"key":{"type":"string","enum":["remote_addr","server_addr","http_x_real_ip","http_x_forwarded_for"]}},"required":["conn","burst","default_conn_delay","key"]},"proxy-cache":{"type":"object","properties":{"cache_bypass":{"type":"array","minItems":1,"items":{"type":"string","pattern":"(^[^\\$].+$|^\\$[0-9a-zA-Z_]+$)"}},"no_cache":{"type":"array","minItems":1,"items":{"type":"string","pattern":"(^[^\\$].+$|^\\$[0-9a-zA-Z_]+$)"}},"hide_cache_headers":{"type":"boolean","default":false},"cache_http_status":{"default":[200,301,404],"minItems":1,"items":{"description":"http response status","minimum":200,"maximum":599,"type":"integer"},"type":"array","uniqueItems":true},"cache_method":{"default":["GET","HEAD"],"minItems":1,"items":{"description":"http method","type":"string","enum":["GET","POST","PUT","DELETE","PATCH","HEAD","OPTIONS","CONNECT","TRACE"]},"type":"array","uniqueItems":true},"cache_zone":{"type":"string","minLength":1},"cache_key":{"type":"array","minItems":1,"items":{"description":"a key for caching","pattern":"(^[^\\$].+$|^\\$[0-9a-zA-Z_]+$)","type":"string"},"default":["$host","$request_uri"]}},"required":["cache_zone"]},"zipkin":{"type":"object","properties":{"sample_ratio":{"type":"number","minimum":1e-05,"maximum":1},"endpoint":{"type":"string"},"server_addr":{"type":"string","pattern":"^[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}$","description":"default is $server_addr, you can speific your external ip address"},"service_name":{"type":"string","default":"APISIX","description":"service name for zipkin reporter"}},"required":["endpoint","sample_ratio"]},"batch-requests":{"type":"object","additionalProperties":false}}}
```

Comment:
What is the purpose of this file? Is it automatically generated?

Reply:
sorry for reply late.I think we had discuss it in mail list.